ssz-rs/src/lib.rs

```diff
@@ -127,6 +127,7 @@ pub mod prelude {
     // with a simple `prelude` import
     pub use crate as ssz_rs;
     pub use ssz_rs_derive::{Serializable, SimpleSerialize};
+    pub use alloy_primitives::U256;
```

Comment:
so this should re-export our type (which just happens to be alloy's `U256`

```suggestion
    pub use uint::U256;
```

(although this line should already be present above somehow)